src/internal.c

```diff
@@ -7569,71 +7569,40 @@ int SendUserAuthBanner(WOLFSSH* ssh)
     return ret;
 }
 
-
-int SendRequestSuccess(WOLFSSH* ssh)
-{
-    byte* output;
-    word32 idx;
-    int ret = WS_SUCCESS;
-
-    WLOG(WS_LOG_DEBUG, "Entering SendRequestSuccess");
-
-    if (ssh == NULL)
-        ret = WS_BAD_ARGUMENT;
-
-    if (ret == WS_SUCCESS)
-        ret = PreparePacket(ssh, MSG_ID_SZ);
-
-    if (ret == WS_SUCCESS) {
-        output = ssh->outputBuffer.buffer;
-        idx = ssh->outputBuffer.length;
-
-        output[idx++] = MSGID_REQUEST_SUCCESS;
-        ssh->outputBuffer.length = idx;
-
-        ret = BundlePacket(ssh);
-    }
-
-    if (ret == WS_SUCCESS)
-        ret = wolfSSH_SendPacket(ssh);
-
-    WLOG(WS_LOG_DEBUG, "Leaving SendRequestSuccess(), ret = %d", ret);
-    return ret;
-}
-
-int SendRequestFailure(WOLFSSH *ssh)
+int SendRequestSuccess(WOLFSSH *ssh, int success)
```

Comment:
Can we treat the success parameter as a boolean? We are currently using it with an error code enumeration (WS_SUCCESS) that is getting complemented (~WS_SUCCESS).

Reply:
Done